### PR TITLE
update with html5 compliant code

### PR DESCRIPTION
**Pull request by grandoc**

### CODE REVIEW

**Comment by frederic34** [comment]
@grandoc 
for print_liste_field_titre you have to transform like this:
before:
print_liste_field_titre('Amount', $_SERVER["PHP_SELF"], 'fa_amount', '', '', 'align="right"', $sortfield, $sortorder);

after:
print_liste_field_titre('Amount', $_SERVER["PHP_SELF"], 'fa_amount', '', '', '', $sortfield, $sortorder, 'right ');

note the space after right!
not doing like that you will have two class

**Comment by frederic34** [comment]
image

File: htdocs/contact/consumption.php [hunk 1]
```diff
@@ -398,14 +398,14 @@
 
     // Titles with sort buttons
     print '<tr class="liste_titre">';
-    print_liste_field_titre('Ref', $_SERVER['PHP_SELF'], 'doc_number', '', $param, 'class="left"', $sortfield, $sortorder);
```
Comment (frederic34) [inline htdocs/contact/consumption.php]: $param, **'',** $sortfield, $sortorder, 'left '

Comment (frederic34) [reply htdocs/contact/consumption.php]: you forgot to let , ''

Comment (grandoc) [reply htdocs/contact/consumption.php]: Done.

File: htdocs/compta/paiement_charge.php [hunk 2]
```diff
@@ -282,18 +282,18 @@
 
 		if ($objp->date_ech > 0)
 		{
-			print "<td align=\"left\">".dol_print_date($objp->date_ech, 'day')."</td>\n";
+			print <td class="left">'.dol_print_date($objp->date_ech, 'day').'</td>'."\n";
```
Comment (frederic34) [inline htdocs/compta/paiement_charge.php]: @grandoc missing ' (single quote) after print 👓

**Pull request merged**